src/api-reference/travel/itinerary/trip/trip-resource.markdown

```diff
@@ -353,44 +366,43 @@ The response returns subset of the elements described in the following tables de
-|EndDateUtc	|dateTime	|The end date of the trip, in UTC. Format: YYYY-MM-DDThh:mm:ss
+|EndDateUtc	|dateTime	|The end date of the trip, in UTC. Format: YYYY-MM-DDThh:mm:ss.
 |IsPersonal	|boolean	|Whether the trip is a Business or Leisure trip. Format: true/false.
 |ProjectName	|string	|The associated project name for the trip. Maximum length: 255 characters.
-|StartDateUtc	|dateTime	|The start date of the trip, in UTC. Format: YYYY-MM-DDThh:mm:ss
-|RuleViolations	|array	|The list of rule violations associated with the itinerary. This parent element contains a RuleViolation child element for each associated rule violation.
-|Status	|string	|The status of the itinerary. One of the following: 0- Confirmed; 1- Ticketed by agent; 2- Canceled
-|Bookings	|array	|A parent element that contains a Booking child element for each booking associated with this itinerary.
+|StartDateUtc	|dateTime	|The start date of the trip, in UTC. Format: YYYY-MM-DDThh:mm:ss.
+|RuleViolations	|array	|The list of rule violations associated with the itinerary. This parent element contains a `RuleViolation` child element for each associated rule violation.
+|Status	|string	|The status of the itinerary. One of the following: 0- Confirmed; 1- Ticketed by agent; 2- Canceled.
+|Bookings	|array	|A parent element that contains a `Booking` child element for each booking associated with this itinerary.
 
 ### Booking Element
-The Booking element contains the following elements:
 
 | Element Name | Data Type |Description
 | --------- | --------- | -------
-|Segments	|array	|List of Segments in this booking. The child elements included in this element vary depending on wheter a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the Segments element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the Segments element contains one or more Air, Car, Hotel, or Ride parent element.
-|Passengers	|array	|Contains a Passenger child element for each included passenger. For the descriptions of these elements, see Passengers Elements
-|RecordLocator	|string	|The unique identifier for a booking. This is often six alphanumeric characters but can have other formats depending on the booking source.
+|Segments	|array	|List of segments in this booking. The child elements included in this element vary depending on whether a TMC, third-party developer, or TripLink supplier is requesting the itinerary details: **For TMCs and third-party developers**, the `Segments` element contains one or more Air, Car, Hotel, Dining, Ride, Rail, Parking, or Travel parent elements. **For TripLink suppliers**, the `Segments` element contains one or more Air, Car, Hotel, or Ride parent element.
+|Passengers	|array	|Contains a `Passenger` child element for each included passenger. For the descriptions of these elements, see Passengers Elements.
```

Comment:
This looks like it used to be a link. There is no table for Passengers elements, but there is a more detailed description for Passengers in the Create a New Trip section, so let's link to that.

Reply:
Done, tweaked the description slightly.